.github/workflows/expo.build.yml

```diff
@@ -10,6 +10,12 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
+env: # Define a shared environment variable for android and ios build.
+  MAPBOX_DOWNLOADS_TOKEN: ${{ secrets.MAPBOX_DOWNLOADS_TOKEN }}
+  EXPO_PUBLIC_CLIENT_URL: ${{ vars.EXPO_PUBLIC_CLIENT_URL }}
+  EXPO_PUBLIC_API_URL: ${{ vars.EXPO_PUBLIC_API_URL }}
+  EXPO_PUBLIC_MAPBOX_ACCESS_TOKEN: ${{ secrets.EXPO_PUBLIC_MAPBOX_ACCESS_TOKEN }}
```

Comment:
@andrew-bierman  These variable must be defined. 
here I'm using some `vars` keyword but if these variables are all defined as secrets just let me know and I will update them.

Reply:
I will define these as secrets just to keep everything in the same place, github secrets will be updated with these values momentarily

Reply:
@andrew-bierman updated.